lib/pages/scan/CompaniesList.dart

```diff
@@ -0,0 +1,59 @@
+import 'package:flutter/material.dart';
+import 'package:pola_flutter/ui/list_item.dart';
+import 'package:url_launcher/url_launcher.dart';
+import 'scan_bloc.dart';
+
+class CompaniesList extends StatelessWidget {
+  CompaniesList(this.state, this.listScrollController);
+
+  final ScanLoaded state;
+  final ScrollController listScrollController;
+
+  @override
+  Widget build(BuildContext context) {
+    WidgetsBinding.instance.addPostFrameCallback((_) {
```

Comment:
Scroll to top kiedy dostaniemy nowy item do listy (nowy item pojawia się na samej górze)

Reply:
JAszcze jakby była to prywatna metoda to byłoby super :)